code/Common/Compression.h

```diff
@@ -48,6 +48,13 @@ namespace Assimp {
 /// @brief This class provides the decompression of zlib-compressed data.
 class Compression {
 public:
+    enum class Format {
+        Binary = 0,
+        ASCII,
+
+        NumFormats,
+        InvalidFormat
+    };
```

Comment:
```suggestion
    enum class Format {
        InvalidFormat -1,
        Binary = 0,
        ASCII,

        NumFormats
    };
```
Keep InvalidFormat out of the range of any future valid formats